src/hub.ts

```diff
@@ -31,9 +36,14 @@ export interface AddressHub {
   handleConnection: (ws: WebSocket) => void
 }
 
+export interface AddressHubOpts {
+  plugins: AddressPlugin[]
+  log?: (text: string) => void
+}
+
 interface PluginRow {
-  /**  Maps from addresses to socketId's */
-  addresses: Map<string, Set<number>>
+  /**  Maps address to a socketIds */
```

Comment:
The previous wording, `/**  Maps from addresses to socketId's */`, was more a grammar.